test/skip-subtypes/generated/BodyComplex/README.md

```diff
@@ -1,88 +1,83 @@
 ## An isomorphic javascript sdk for - AutoRestComplexTestService
 
-This package contains an isomorphic SDK for AutoRestComplexTestService.
+This package contains an isomorphic SDK (runs both in node.js and in browsers) for AutoRestComplexTestService.
 
 ### Currently supported environments
 
-- Node.js version 6.x.x or higher
+- Node.js version 8.x.x or higher
 - Browser JavaScript
 
 ### How to Install
 
+To use this SDK in your project, you will need to install two packages.
+- `` that contains the client.

```

Comment:
Should we have some place holder?

Reply:
I used the following as a place holder: "this package that contains the client".